project_documentation/Definition_of_done.txt

```diff
@@ -0,0 +1,24 @@
+Definition of Done:
+
+a User Story (US) is considered Done when all of the following are fullfilled:
+* The assigned reviewer(s) has checked that all the Acceptance Criterias are fullfilled, if there are any Acceptance Criterias
+* All the tasks related to the US are Done
+* The reviewer(s) has tested the US functionality in Android studio and not noticed any errors
+* if the US functionality as a whole was branched out, the Pull Request process has been completed 
```

Comment:
I don't quite understand this line, can you elaborate?

Reply:
Not sure I understand line 7

Reply:
"if the US functionality as a whole was branched out, the Pull Request process has been completed "
What I tried to say was:
If we choose to make a seperate branch for not just a task but instead for the whole User Story, 
then we should complete the Pull Request process (as it is defined on line 14 and onward), before we can tick this part of the Definition of Done.

Did I become more clear, @antonsvaren  and @eliascalzone ?